VMwarePSDesiredStateConfiguration.md

```diff
@@ -1,30 +1,21 @@
 ## Getting Started
 
-**VMware.PSDesiredStateConfiguration** provides a new way to compile and execute **DSC Configurations** on **Windows**, **Linux** and **MacOS** that does not require the use of **DSC LCM** or the typical MOF format and without altering the existing **Configuration** syntax.  Instead the cmdlets of this module create and work with regular powershell objects. This module is also designed to work with the existing **DSC Resources** in **VMware.vSphereDSC**.  
+**VMware.PSDesiredStateConfiguration** provides commands to compile and execute **DSC Configuration** without using the DSC Local Configuration Manager. Compiled configuration is stored in memory as PS Object. Set, Get, Test-... cmdlets use Invoke-DSCResource cmdlet of PSDesiredStateConfiguration module to execute/run/process the compiled configuration. This solution allows cross platform support on MacOS and Linux. This module is also designed to work with the existing **DSC Resources** in **VMware.vSphereDSC**.
```

Comment:
Complete the name of the Set, Get , Test cmdlets. I didn't do that in the comment because it was easier to me and I was expecting you to make it :)
execute/run/process -> pick up one of those